templates/item/macros.j2

```diff
@@ -103,16 +113,55 @@
 {% endmacro %}
 
 
-{% macro summary(content) %}
-  {% if content %}
+{% macro summary(item) %}
+  {% set content = item.description %}
+  {% set origin = item.since %}
+  {% set by = item.author %}
+  {% set deprecation = item.deprecated %}
+
+  {% if content or origin or by or deprecation %}
     <div data-item-section="summary">
       <h3 class="item-subtitle">Description</h3>
-      {{ content|safe }}
+
+      <div class="text-block">
+        {{ content|safe }}
+        {{ since(origin) }}
+        {{ authors(by) }}
+        {{ deprecated(deprecation) }}
+      </div>
     </div>
   {% endif %}
 {% endmacro %}
 
 
+{% macro authors(data) %}
+  {% if (data.length > 0) %}
+    <h4 class="summary-subtitle">Authors</h4>
```

Comment:
Sorta weird to see Authors (plural) and only one name. Again, maybe I am looking at this too carefully.